.eslintignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+reports
+public/js/bundle.js
+acceptance_tests
+dist/js/bundle.js
+acceptance_tests
+doc/**/*
+apps/**/translations/**/default.json
```

Comment:
Not all of these things are in this repo. Remove the items that aren't in the repo